### DISCUSSION TITLE
x,y positions are not updated

### DISCUSSION THREAD

**Discussion by Felix-kkk** [bitcraze/discussions#1653] [Q&A] [answered]
I call cfclient and find that x,y has no data and can't control it
![image](https://github.com/user-attachments/assets/b9650562-05fa-4a63-82c2-6eb668c595df)

**Comment by knmcguire** [comment, marked as answer]
Hi! This is what happens when the complimentary filter is selected, which is usually the case if no 'positioning deck' is attached. 

Could you help out with the following (these are from te [discussion rules](https://github.com/orgs/bitcraze/discussions/55))

> Provide as much information about your problem as you can. The [cfclients](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/)'s debug tab (found under 'Help'>'about') can already provide handy information about your development environment and your connected Crazyflie. Copy pasting the[ console tab's content](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/console_tab/) also helps a lot.

And also which type of positioning are you using?
